cmd/release-notes/main.go

```diff
@@ -262,6 +268,13 @@ func GetReleaseNotes() (notes.ReleaseNotes, notes.ReleaseNotesHistory, error) {
 	return releaseNotes, history, nil
 }
 
+func or(opt, def string) string {
```

Comment:
```suggestion
func or(opt, default string) string {
```

Let's add a function comment here.

Reply:
`default` is a reserved word: https://golang.org/ref/spec#Keywords.
However, that func is actually not needed, as we already do defaulting on the `opts` struct.